src/Evolution/DiscontinuousGalerkin/SlopeLimiters/MinmodType.cpp

```diff
@@ -10,14 +10,14 @@
 #include "Options/Options.hpp"
 #include "Options/ParseOptions.hpp"
 
-std::ostream& SlopeLimiters::operator<<(
```

Comment:
An FYI can be done later, the stream operator should be in the `Limiters` namespace for ADL to work correctly. As it stands, there will be situations where the stream operator won't be found because only the `std` and `Limiters` namespaces will be searched. I'll leave it up to you to decide whether to deal with this now or when/if it happens :)

Reply:
Ah, right. I'll do this in a subsequent PR once this one is merged :)